alignak_module_backend/arbiter/module.py

```diff
@@ -648,19 +646,19 @@ def get_services(self):
         :return: None
         """
         self.configraw['services'] = {}
-        params = {'embedded': '{"escalations":1,"service_dependencies":1}',
+        params = {'sort': 'host', 'embedded': '{"host":1,"escalations":1,"service_dependencies":1}',
```

Comment:
Ok, but if you have a better idea to get the name of the host the service is attached to  ???